src/material/schematics/ng-generate/navigation/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.ts.template

```diff
@@ -26,7 +27,8 @@ import { map, shareReplay } from 'rxjs/operators';
     MatButtonModule,
     MatSidenavModule,
     MatListModule,
-    MatIconModule
+    MatIconModule,
+    CommonModule
```

Comment:
This should import the `AsyncPipe` from `@angular/common`, rather than the `CommonModule`.

Reply:
But also the NgIf is used.
Should we add it or just use the CommonModule as usual?

Reply:
We should do `NgIf` since we don't need the entire module.

Reply:
Code updated as discussed